### PR TITLE
Copyright header fix

### PR DESCRIPTION
**Pull request by cagryInside**

### CODE REVIEW

File: src/main/java/edu/usf/cutr/trackerlib/location/DouglasPeuckerDecimation.java [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright 2008 Google Inc., 2015 Cagri Cetin (cagricetin@mail.usf.edu),
- * University of South Florida
+ * Copyright 2008 Google Inc., 2015 University of South Florida (cagricetin@mail.usf.edu)
```
Comment (barbeau) [inline src/main/java/edu/usf/cutr/trackerlib/location/DouglasPeuckerDecimation.java:2]: @cagryInside Could you change this to `Copyright 2008-2015 Google Inc., University of South Florida (cagricetin@mail.usf.edu)`?

This is the more common format I've seen with date ranges.

Comment (cagryInside) [inline src/main/java/edu/usf/cutr/trackerlib/location/DouglasPeuckerDecimation.java:2]: @barbeau Sure.

File: src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <resources>
-    <!-- Copyright (C) 2015 Cagri Cetin (cagricetin@mail.usf.edu), University of South Florida
+    <!-- Copyright (C) 2015 University of South Florida (cagricetin@mail.usf.edu)
     and individual contributors.
```
Comment (barbeau) [inline src/main/res/values/strings.xml:3]: Let's remove `and individual contributors.` and replace them with the known contributors, if we know this.  I started including this wording in OBA Android, but in retrospect it was a mistake.  We should try to list all known authors.

File: src/main/java/edu/usf/cutr/trackerlib/utils/TimeUtils.java [hunk 3]
```diff
@@ -25,7 +25,7 @@ public static long createBatchUpdateTimeMillis() {
         //Add two hours
         long twoHourMillis = 2*60*60*1000;
         // Test data
-        twoHourMillis = (60 * 1000);
+//        twoHourMillis = (60 * 1000);
```
Comment (barbeau) [inline src/main/java/edu/usf/cutr/trackerlib/utils/TimeUtils.java:28]: Is this intended to be commented out?

**Comment by barbeau** [comment]
Thanks @cagryInside!  A few comments inline.

File: src/main/java/edu/usf/cutr/trackerlib/utils/TimeUtils.java (hunk 3 above)
Comment (cagryInside) [inline src/main/java/edu/usf/cutr/trackerlib/utils/TimeUtils.java:28]: Yes, this was for test purposes (schedules batch updates every minute).  Maybe, I should put this line under `if (BuildConfig.DEBUG) { ... }` .

Comment (barbeau) [inline src/main/java/edu/usf/cutr/trackerlib/utils/TimeUtils.java:28]: > Maybe, I should put this line under if (BuildConfig.DEBUG) { ... }

Good idea!

**Pull request merged**